spec/06-expressions.md

```diff
@@ -261,13 +261,13 @@ If ´f´ has some value type, the application is taken to be equivalent to `´f
 i.e. the application of an `apply` method defined by ´f´. The value `´f´` is applicable to the given arguments if `´f´.apply` is applicable.
 
 
-Evaluation of `´f´(´e_1 , \ldots , e_n´)` usually entails evaluation of
-´f´ and ´e_1 , \ldots , e_n´ in that order. Each argument expression
-is converted to the type of its corresponding formal parameter.  After
-that, the application is rewritten to the function's right hand side,
-with actual arguments substituted for formal parameters.  The result
-of evaluating the rewritten right-hand side is finally converted to
-the function's declared result type, if one is given.
+The application `´f´(´e_1 , \ldots , e_n´)` evaluates ´f´ and then each argument
+´e_1 , \ldots , e_n´ from left to right, unless it's a by name parameter (see
+below).  Each argument expression is converted to the type of its corresponding formal parameter.
+After that, the application is rewritten to the function's right hand side, with
+actual arguments substituted for formal parameters.  The result of evaluating
```

Comment:
Are those double spaces intentional? And are they rendered as such?

```suggestion
below). Each argument expression is converted to the type of its corresponding formal parameter.
After that, the application is rewritten to the function's right hand side, with
actual arguments substituted for formal parameters.  he result of evaluating
```

Reply:
There already were double spaces and I kept it that way. I assume they're not rendered.

Reply:
The text was already double-spaced. I kept in the same style. I assume they're not rendered differently than single spaces.

Whether there should be one or two spaces behind a period is, I believe, the tabs vs spaces of typesetting.

Reply:
It's the new "generation gap". Someone just emailed me that she tries to remember not to double space because it "dates" her. Do we mind if Martin double spaces? I agree it doesn't matter with a modern renderer.

Reply:
Note that if you're using a typewriter, you can't white-out a double space.

Reply:
"Not a typewriter"